dolphin/common/fields.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Custom fields for Dolphin objects."""
+
+
+class StorageStatus(object):
```

Comment:
enum or object here?

Reply:
'object' will ensure status is still String type. 'enum' requires changes in db model, serialization etc.